zbuf/bufsort.go

```diff
@@ -0,0 +1,60 @@
+package zbuf
+
+import (
+	"context"
+
+	"github.com/brimsec/zq/expr"
+)
+
+type BufferedSort struct {
```

Comment:
```suggestion
// BufferedSort reads from its reader, and accumulates records into a batch until either it reaches a maximum batch size, a timeout expires, or the stream ends. At that point, the batch is sorted and returned.
type BufferedSort struct {
```

Ok I realized that `Batch()` has the documentation  and since this a single-method struct, that's probably ok.